packages/ui/src/composites/wui-input-text/styles.ts

```diff
@@ -102,6 +101,17 @@ export default css`
     left: var(--wui-spacing-l);
   }
 
+  .wui-size-mdl {
+    padding: 17.25px var(--wui-spacing-m) 17.25px var(--wui-spacing-m);
```

Comment:
If these kind of values come from the design file, I think we should ask why we don't have regular spacing values. Because sometimes, the designer might miss some values and those could be unintentional. 

We should be following the practice of 2x (or similar) spacing system

Reply:
I agree, I think this happens in other places in our composites/components as well. The buttons and `wui-input-email` have a height of `56px`, changing these values will change this. So I think we should keep it as it is and revisit when we will split the ui package as part of the bigger refactor.